content/en/apps/reference/app-settings/token_login.md

```diff
@@ -0,0 +1,31 @@
+---
+title: ".token_login"
+linkTitle: ".token_login"
+weight: 5
+description: >
+  **Token login**: Instructions and schema for Login by SMS
+relatedContent: >
+  apps/reference/api#login-by-sms
+ 
+keywords: sms login users
```

Comment:
We ended up switching to show links to the explicit relatedContent links rather than figuring it out from the keywords. It doesn't hurt to have the keywords, but they won't be used in the site.

Reply:
I've removed keywords.